src/main/java/ListUtils.java

```diff
@@ -0,0 +1,155 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class ListUtils {
+    //class to store task
+    private static class Task {
+        String description;
+        boolean isDone;
+        Task(String description) {
+            this.description = description;
+            this.isDone = false;
+        }
+        void markAsDone() {
+            this.isDone = true;
+        }
+
+        void markAsNotDone() {
+            this.isDone = false;
+        }
+
+        @Override
+        public String toString() {
+            return "[" + (isDone ? "X" : " ") + "] " + description;
+        }
+
+    }
+
+    private static class Todo extends Task {
+        Todo(String description) {
+            super(description);
+        }
+
+        @Override
+        public String toString() {
+            return "[T]" + super.toString();
+        }
+    }
+
+    private static class Deadline extends Task {
+
+        protected String by;
+
+        public Deadline(String description, String by) {
+            super(description);
+            this.by = by;
+        }
+        @Override
+        public String toString() {
+            return "[D]" + super.toString() + " (by: " + by + ")";
+        }
+    }
+
+    private static class Event extends Task {
+        protected String from;
+        protected String to;
+        Event(String description, String from, String to) {
+            super(description);
+            this.from = from;
+            this.to = to;
+        }
+
+        @Override
+        public String toString() {
+            return "[E]" + super.toString() + " (from: " + from + ", to: " + to + ")";
+        }
+
+    }
+    // create list of tasks
+    private static List<Task> userInputs = new ArrayList<>();
+
+    //create methods to add each respective task subclasses
+
+    public static void addTodoToList(String description) {
```

Comment:
do you feel like you are repeating a lot of lines, for adding to the list, is there a workaround this